lib/src/components/NxTabs/NxTab.tsx

```diff
@@ -40,6 +47,44 @@ const NxTab = function NxTabElement(props: NxTabProps) {
     }
   }
 
+  function handleKeyDown(event: React.KeyboardEvent<HTMLLIElement>) {
+    if (event.isDefaultPrevented()) {
+      return;
+    }
+
+    const tabElements = event.currentTarget.parentElement?.children;
```

Comment:
The code below is written as if there's a possibility that this is `null`ish, but really that can't happen right? I guess if the `<li>` was somehow the root of the document it could, but that shouldn't ever be the case.  Might be worthwhile to use the non-nullish assertion operator (`!`) so that the code below can be simplified.

Reply:
> The code below is written as if there's a possibility that this is `null`ish, but really that can't happen right? I guess if the `<li>` was somehow the root of the document it could, but that shouldn't ever be the case. Might be worthwhile to use the non-nullish assertion operator (`!`) so that the code below can be simplified.

Thanks, cleaned up here: 7dbc588c766bf52503c345f23530850a30eec371